Update wallet documentation (#59)
MetaMask/mm-docs-v2: wallet/how-to/use-mobile.md

}
```

> The extension provider is always available by the time your code is executed.

Reviewer: ```suggestion
The extension provider is always available when your code executes.
```

## Test your dapp

To test your dapp's integration with MetaMask Mobile:

1. Configure your development server to run on your host machine's local IP address `192.168.x.x`,
    or `0.0.0.0`.

> 1. Make sure your testing device uses the same WiFi connection as the machine hosting the server.

Reviewer: ```suggestion
1. Ensure your testing device uses the same WiFi connection as the machine hosting the server.
```
